common/v2/features/SendAssets/components/SendAssetsForm.tsx

```diff
@@ -274,14 +279,24 @@ const SendAssetsForm = ({ txConfig, onComplete }: IStepComponentProps) => {
         onSubmit={fields => {
           onComplete(fields);
         }}
-        render={({ errors, setFieldValue, setFieldTouched, touched, values, handleChange }) => {
+        render={({ errors, setFieldValue, setFieldTouched, touched, values }) => {
           if (getProTxValue(['setMainTransactionFormCallback'])) {
             getProTxValue(['setMainTransactionFormCallback'])(() => ({
               isValid: isFormValid,
               values
             }));
           }
 
+          // Set gas estimates if default asset is selected
+          useEffectOnce(() => {
+            if (!isEmpty(values.asset)) {
+              fetchGasPriceEstimates(values.network).then(data => {
+                setFieldValue('gasEstimates', data);
+                setFieldValue('gasPriceSlider', data.fast);
+              });
+            }
+          });
```

Comment:
DANGER! Hooks should only be called at the top level. This is an unsupported behaviour https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level